File: week01/MajorityElement.java
public int majorityElement(int[] nums) {
        int voteCount = 1;
        int winner = nums[0];
        for(int i=1; i<nums.length; i++){
            if(nums[i] != winner){
                voteCount--;                                      
                if(voteCount == -1){
                  winner= nums[i];
                    voteCount = 1;
                } 
            }else voteCount++;
        }
        return winner;
    }

File: week01/majorityElement.js
var majorityElement = function (nums) {
	let winner = 0;
	let vote_count = 0;

	for (let i = 0; i < nums.length; i++) {
		if (vote_count == 0) {
			winner = nums[i];
		}
		if (nums[i] == winner) {
			vote_count++;
		} else {
			vote_count--;
		}
	}
	return winner;
};
